playground/tanka/apps/aperture-go-example/main.jsonnet

```diff
@@ -0,0 +1,15 @@
+local mixins = import 'mixins.libsonnet';
+
+function(apiServer='API SERVER MISSING') {
+  apiVersion: 'tanka.dev/v1alpha1',
+  kind: 'Environment',
+  metadata: {
+    name: 'environment/tilt/apps/aperture-go-example',
```

Comment:
It should be: `apps/aperture-go-example` as we don't have `environment/tilt` in aperture.